src/main/configuration/am.properties

```diff
@@ -24,7 +24,7 @@ archivesDirectoryName=archives
 catalogsCacheDirectoryName=catalogs
 localAddonsCatalogFilename=local.json
 overwrittenFilesDirectoryName=overwritten
-remoteCatalogUrl=http://www.exoplatform.com/addons/catalog
+remoteCatalogUrl=https://raw.githubusercontent.com/Meeds-io/meeds/catalog/catalog
```

Comment:
It means the tooling has been updated to publish the catalog here now ?

Reply:
nope, this is a placeholder to show that now we can configure an URL for Meeds and another for eXo EE. It will be replaced by a good URL as soom as provided by @vsellier

Reply:
ok